examples/tutorial/00_Setup.ipynb

```diff
@@ -87,11 +87,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import pandas\n",
-    "import datashader\n",
+    "import pandas as pd\n",
+    "import datashader as ds\n",
     "import dask\n",
-    "import geoviews\n",
-    "import bokeh"
+    "import geoviews as gv\n",
+    "import bokeh\n",
+    "\n",
+    "from distutils.version import LooseVersion\n",
+    "assert LooseVersion(pd.__version__) >= LooseVersion('0.24.0')\n",
+    "assert LooseVersion(ds.__version__) >= LooseVersion('0.6.9')\n",
+    "assert LooseVersion(dask.__version__) >= LooseVersion('1.1.0')\n",
+    "assert LooseVersion(gv.__version__) >= LooseVersion('1.6.2')\n",
+    "assert LooseVersion(dask.__version__) >= LooseVersion('1.0.4')"
```

Comment:
Is this too verbose for you @jbednar

Reply:
Yup.

Reply:
Would you be happier if versions were just printed?

Reply:
I was thinking of adding `pyviz info` like `bokeh info` but I don't want to mandate that people have the package `pyviz`.

Reply:
Maybe:

```
import dask, geoviews as gv
from distutils.version import LooseVersion

required_versions = dict(ds='0.6.9', dask='1.1.0', gv='1.6.2')

for lib, ver in required_versions.items():
    assert LooseVersion(getattr(globals()[lib],'__version__')) >= LooseVersion(ver)
```

Reply:
oh so just slightly too verbose :)

Reply:
Printing the version in that loop would be fine too.  We should talk about this, I already have code for checking versions that I want to make sure to make available *somewhere*, and we may as well have it support this usage as well.

Reply:
(I mean *in addition* to checking it.  I don't care whether it's an assertion or some other form of error message, but I do think it should be an error rather than continuing on successfully.)

Reply:
Ok in that case I don't think printing is very helpful.